ctapipe/ml/sklearn.py

```diff
@@ -29,9 +29,16 @@ class Model(Component):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        self.model = SUPPORTED_MODELS[self.model_cls](**self.model_config)
+
+        # to verify settings
+        self.new_model()
```

Comment:
what kind of exception would this raise? something that is easily understandable? or should we use a `try` block with some kind of `BrokenConfigException`?

Reply:
Either a key error for an unsupported model or a sklearn error im case of unknown options, the ones I found were pretty clear

Reply:
Great!